server/main/templates/account/login.html

```diff
@@ -6,12 +6,16 @@
 {% load static %}
 
 {% block head_title %}
-  {{ block.super }}
+  {{ block.super }} &mdash; {% translate 'Login' %}
   {% if form.non_field_errors|length %}
-    {% translate '- Login Error' %}
+    {% translate ' Error' %}
```

Comment:
Splitting up translation strings around control flow like this is not a good idea. It works in English, but not necessarily in other languages where the grammar puts the parts of speech in a different order, or if there's an entirely different word/words to use in the different contexts. So instead, this would be

```
{% if form.non_field_errors|length %}
  {{ block.super }} &mdash; {% translate "Login" context "page title" %}
{% else %}
  {{ block.super }} &mdash; {% translate "Login Error" context "page title" %}
{% endif %}